src/create.js

```diff
@@ -197,6 +201,10 @@ function newThingUI (context, thePanes) {
   }
 }
 
+function validatePathname (pathname) {
+  return !(/[#:]/g).test(pathname)
```

Comment:
I'd avoid regular expressions if possible, since they're relatively slow (not really a problem outside of hot code paths, such as this) and often hard to read. In this case, I'd just go for two `indexOf`'s.
```suggestion
  return pathname.indexOf('#') === -1 && pathname.indexOf(':') === -1
```